### DISCUSSION TITLE
Sequential Wildcards?

### DISCUSSION THREAD

**Discussion by mike4llison** [adieyal/sd-dynamic-prompts#72] [Q&A] [answered]
Hello all, is there a way to force dynamic prompts to get a name from a wildcard list and not repeat it? Say for instance I have a list of 100 artists and I want to see what they all look like with the same prompt and seed? I am using fixed seed in the advanced panel but there are so many repeating wildcards.

Thanks

M.

**Comment by adieyal** [comment]
I'll look into it, you should only see repeats if your quantity is less than the number of variants, e.g. {5$$red|green|blue}. As an alternative, have you tried combinatorial mode? That will produce 100 images, one image per artists

**Reply by mike4llison** [reply]
Thanks for the reply, I did try combinatorial mode but it didn't really seem to fit my needs and kept getting interrupted with a huge spurious error output with `IndexError: list index out of range`
Initially, I was trying to see precisely how a wildcard list of people looks with the same prompt and seed. Your extension is the perfect vessel for that If I could figure that out. It was a list of about 300 but I didn't really want the full 300. I tried cutting down the list to 50 and there were just as many repeats.

**Comment by adieyal** [comment, marked as answer]
Coming back to this now - I think I misunderstood your original comment. The correct way to do this is to use combinatorial mode and then set the maximum number of images (e.g. 100) by setting your num batches and batch size appropriately. If you're encountering an error doing that, please open an issue and post the complete error message for me to look at.
